packages/design-picker/tsconfig.json

```diff
@@ -1,34 +1,11 @@
 {
+	"extends": "@automattic/calypso-build/typescript/ts-package.json",
 	"compilerOptions": {
-		"target": "ES5",
-		"lib": [ "DOM", "ES2015" ],
-
-		"baseUrl": ".",
-		"module": "esnext",
-		"allowJs": false,
-		"jsx": "react",
-		"declaration": true,
 		"declarationDir": "dist/types",
 		"outDir": "dist/esm",
 		"rootDir": "src",
-
-		"strict": true,
-		"noUnusedLocals": true,
-		"noUnusedParameters": true,
-		"noImplicitReturns": true,
-		"noFallthroughCasesInSwitch": true,
-
-		"moduleResolution": "node",
-		"esModuleInterop": true,
 		"resolveJsonModule": true,
-		"forceConsistentCasingInFileNames": true,
-
-		"typeRoots": [ "../../node_modules/@types" ],
-		"types": [],
-
-		"importsNotUsedAsValues": "error",
-		"importHelpers": true,
-		"composite": true
+		"types": []
```

Comment:
Why'd we keep this one?

Reply:
Because otherwise this package fails.

With the base `types: ["node"]` I get
```
src/components/mshots-image/index.tsx:122:5 - error TS2322: Type 'Timeout' is not assignable to type 'number'.

122     timeoutIdRef.current = setTimeout( () => setCount( ( count ) => count + 1 ), count * 500 );
        ~~~~~~~~~~~~~~~~~~~~
```

When using node types, `setTimeout()` returns a `NodeJS.Timeout` (https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node/globals.d.ts#L50). However, when using DOM types (https://github.com/microsoft/TypeScript/blob/main/lib/lib.dom.d.ts#L19627), it returns a `number`.

I figured it was better to drop the node types than using `NodeJS.Timeout` types in a browser package.